packages/botonic-ci/README.md

```diff
@@ -0,0 +1,56 @@
+# Botonic CI (continuous integration)
+
+## What Does This Package Do?
+
+This package simplifies the configuration of all CI tasks for projects which use botonic.
+
+### Features
+
+* eslint
+* typescript type checking
+
+
+## Setup
+
+* Install this package
+```
+npm run -D @botonic/ci
```

Comment:
should this be `npm install -D @botonic/ci` ?

Reply:
fixed, thanks